storage/process_updates.go

```diff
@@ -44,15 +47,55 @@ func ProcessUpdateCallbackSetStatus(status types.ProcessStatus) func(*types.Proc
 	}
 }
 
-// ProcessUpdateCallbackAcceptingVotes returns a function that updates the accepting votes flag
+// ProcessUpdateCallbackSetStateRoot returns a function that updates the state
+// root and vote counts of a process. This function is used when a update over
+// the state root is received from the process monitor.
+func ProcessUpdateCallbackSetStateRoot(newRoot *types.BigInt, newCount, newOverwrittenCount *types.BigInt) func(*types.Process) error {
+	// By the moment, do not update the state root, just the vote counts.
```

Comment:
So, if we don't update the state root, what's the current expected behavior if another Sequencer sends a transition state?